test/addons/rook-operator/kustomization.yaml

```diff
@@ -3,29 +3,19 @@
 
 # yamllint disable rule:line-length
 ---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
 resources:
   - https://raw.githubusercontent.com/rook/rook/release-1.13/deploy/examples/crds.yaml
   - https://raw.githubusercontent.com/rook/rook/release-1.13/deploy/examples/common.yaml
   - https://raw.githubusercontent.com/rook/rook/release-1.13/deploy/examples/operator.yaml
-patchesJson6902:
-  - target:
-      kind: ConfigMap
-      name: rook-ceph-operator-config
-      namespace: rook-ceph
-    patch: |-
-      - op: add
-        path: /data/CSI_ENABLE_CSIADDONS
-        value: 'true'
-      - op: add
-        path: /data/CSI_ENABLE_OMAP_GENERATOR
-        value: 'true'
 
 patches:
   # Disable to avoid random failures when restaring the enviroment.
-  #   failed to call webhook:
-  #   Post "https://rook-ceph-admission-controller.rook-ceph.svc:443/
-  #   validate-ceph-rook-io-v1-cephblockpool?timeout=5s":
-  #   x509: certificate signed by unknown authority
+  # failed to call webhook:
+  # Post "https://rook-ceph-admission-controller.rook-ceph.svc:443/
+  # validate-ceph-rook-io-v1-cephblockpool?timeout=5s":
+  # x509: certificate signed by unknown authority
```

Comment:
You change the previous indentation, is this intentional? You did not mention this change in the commit message, so I guess this is unintended change.